streams/src/main/java/org/apache/kafka/streams/StreamsBuilder.java

```diff
@@ -212,7 +212,7 @@
      * {@link KafkaStreams#store(StoreQueryParameters) KafkaStreams#store(...)}:
      * <pre>{@code
      * KafkaStreams streams = ...
-     * ReadOnlyKeyValueStore<K,ValueAndTimestamp<V>> localStore = streams.store(queryableStoreName, QueryableStoreTypes.<K, ValueAndTimestamp<V>>timestampedKeyValueStore());
+     * ReadOnlyKeyValueStore<K,ValueAndTimestamp<V>> localStore = streams.store(queryableStoreName, QueryableStoreTypes.<K, ValueAndTimestamp<V>> timestampedKeyValueStore());
```

Comment:
I think @ableegoldman meant in the return type:
```
ReadOnlyKeyValueStore<K,ValueAndTimestamp<V>> localStore
// -> should be
ReadOnlyKeyValueStore<K, ValueAndTimestamp<V>> localStore
```

The space you inserted should _not_ be there.

Reply:
thanks i updated!